uiowa/uiowa
# UI Press updates

Resolves #5400 

# How to test
- `ddev blt ds --site uipress.uiowa.edu`
- `ddev drush @uipress.local uli /`
- Visit https://uipress.uiowa.ddev.site/seasons and observe 3 columns and typography change.
- Visit https://uipress.uiowa.ddev.site/series and observe 2 columns and typography change.
- Visit https://uipress.uiowa.ddev.site/subjects and observe 3 columns and typography change.
- Visit https://uipress.uiowa.ddev.site/books/sale. Spot check books to make sure that they are displaying a sale price and that they should be on sale.
